gridstatus/spp.py

```diff
@@ -704,6 +704,56 @@ def _process_operating_reserves(self, df):
 
         return df
 
+    @support_date_range("DAY_START")
+    def get_day_ahead_marginal_clearing_prices(self, date, end=None, verbose=False):
+        """Provides Marginal Clearing Price information by Reserve Zone for each
+        Day-Ahead Market solution for each Operating Day.
+        Posting is updated each day after the DA Market results are posted.
+        Available at https://portal.spp.org/pages/da-mcp#
+
+        Args:
+            date: date to get data for
+            end: end date
+            verbose: print url
+
+        Returns:
+            pd.DataFrame: Day Ahead Marginal Clearing Prices
+        """
+        if date == "latest":
+            raise ValueError(
+                "Latest not supported for Day Ahead Marginal Clearing Prices",
+            )
+
+        url = f"{FILE_BROWSER_DOWNLOAD_URL}/da-mcp?path=/{date.strftime('%Y')}/{date.strftime('%m')}/DA-MCP-{date.strftime('%Y%m%d')}0100.csv"  # noqa
+
+        msg = f"Downloading {url}"
+        log(msg, verbose)
+        df = pd.read_csv(url)
+
+        return self._process_day_ahead_marginal_clearing_prices(df)
+
+    def _process_day_ahead_marginal_clearing_prices(self, df):
+        df = self._handle_market_end_to_interval(
+            df,
+            column="GMTIntervalEnd",
+            interval_duration=pd.Timedelta(hours=1),
+        )
+
+        column_mapping = {
+            "RegUP": "Reg_Up",
+            "RegDN": "Reg_Dn",
+            "RampUP": "Ramp_Up",
+            "RampDN": "Ramp_Dn",
+            "UncUP": "Unc_Up",
+            "Spin": "Spin",
+            "Supp": "Supp",
+        }
+
+        return df.rename(columns=column_mapping)[
+            ["Time", "Interval Start", "Interval End", "Reserve Zone"]
```

Comment:
`get_operating_reserves` keeps an `Interval` column from the original data. That seemed extraneous to me with "Interval Start" and "Interval End" so I didn't keep it here.

Reply:
you're right. i caught that after merging. just add this PR: https://github.com/kmax12/gridstatus/pull/318